generators/cleanup.js

```diff
@@ -487,6 +487,12 @@ function cleanupOldServerFiles(generator, javaDir, testDir, mainResourceDir, tes
   if (generator.isJhipsterVersionLessThan('7.0.0-beta.1')) {
     generator.removeFile(`${javaDir}config/CloudDatabaseConfiguration.java`);
   }
+
+  if (generator.isJhipsterVersionLessThan('7.4.1')) {
+    generator.removeFile(`${testResourceDir}cucumber.properties`);
+    generator.removeFile(`${testDir}../features/gitkeep`);
+    generator.removeFile(`${testDir}../features/user/user.feature`);
+  }
```

Comment:
Move to https://github.com/jhipster/generator-jhipster/blob/9e0f409a24262aaf85959529fc6ffb29b6e29635/generators/server/index.js#L440-L441

Like:
```
    cleanupCocumberTests() {
      if (!this.cocumberTests) return undefined;
      this.removeFile(`${this.testResourceDir}cucumber.properties`);
      this.removeFile(`${this.testDir}../features/gitkeep`);
      this.removeFile(`${this.testDir}../features/user/user.feature`);
    }
```

For better modularization.

Reply:
Even better, handling SRP with the help of different files ;-)